src/components/router/FallbackRoute.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useMemo } from 'react';
+import { Navigate, useLocation } from 'react-router-dom';
+
+import Page from 'components/Page';
+import globalize from 'lib/globalize';
+import LinkButton from 'elements/emby-button/LinkButton';
+
+const FallbackRoute = () => {
+    const location = useLocation();
+
+    // Check if the requested path should be redirected
+    const to = useMemo(() => {
+        const _to = {
+            search: location.search,
+            hash: location.hash
+        };
+
+        // If a path ends in ".html", redirect to the path with it removed
+        if (location.pathname.endsWith('.html')) {
+            return { ..._to, pathname: location.pathname.slice(0, -5) };
+        }
+    }, [ location ]);
+
+    if (to) {
+        console.warn('[FallbackRoute] You are using a deprecated URL format. This will stop working in a future Jellyfin update.');
+
+        return (
+            <Navigate
+                replace
+                to={to}
+            />
+        );
+    }
+
+    return (
+        <Page
+            id='fallbackPage'
+            className='mainAnimatedPage libraryPage'
+        >
```

Comment:
```suggestion
        <Page
            id='fallbackPage'
            title={globalize.translate('HeaderPageNotFound')}
            className='mainAnimatedPage libraryPage'
        >
```

Maybe we can add the title?